testing/ods/jh-at-scale.sh

```diff
@@ -211,7 +211,9 @@ run_multi_cluster() {
                      "$ODS_CI_USER_PREFIX" "$ODS_CI_NB_USERS" \
                      "$S3_LDAP_PROPS" \
                      --sut_cluster_kubeconfig="$KUBECONFIG_SUTEST" \
-                     --artifacts-collected=$collect
+                     --artifacts-collected=$collect \
+                     --ods_sleep_factor="$ODS_SLEEP_FACTOR" \
+                     --ods_ci_exclude_tags Notebook
```

Comment:
@fcami this should be removed (otherwise only the authentication is done)

Reply:
Good catch, pushed